docs/getting-involved-with-sourcecred.md

```diff
@@ -0,0 +1,89 @@
+
+This document is less a step-by-step guide, and more an explanation of what it looks like to go from being a newcomer to a full-fledged contributor within the SourceCred community accompanied by the resources that will help you get there. Experience these phases on your own timeline; no phase of getting to know SourceCred is any better or worse than another. This is your experience, and we encourage you to engage with it to the level that fits you.
+
+So, how does one "get involved" with SourceCred? ;)
+
+\[Blush gif? ^_^]
+
+# Some General Tips:
+
+SourceCred is still young, and we don't have many automated onboarding tools yet. That means you'll do best if are willing to dive in, ask questions, explore resources, and otherwise utilize self-directed learning. However, this doesn't mean we're throwing you to the sharks in a 'sink or swim' kind of situation. What we lack in polished or automated onboarding flows, we make up for in high community engagement. This means that your best chance at getting up to speed with what's going on and how to help is to get connected to the community. Come to calls, talk to contributors with more context, ask questions, and offer your help. We'll be glad to welcome you.
+
+Note that just as everyone is different, everyone's journey through SourceCred will be different. Make it the experience you want to have. Go at the pace that helps you realize your goals and build trust in both directions. Below there are helpful suggestions or "quests" for each phase, but none of these are required or need to be done in a specific order. Happy exploring!
+
+# Phase 1: Getting Plugged In
+
+The first step to joining our internal community is to get connected with the tools and social norms we use to communicate and coordinate with each other. In this phase, the focus is on getting you plugged into the different platforms, resources, and expectations of our community. This will give you a good foundation for the exploration you'll do if you decide to contribute.
+
+If you decide that you don't want to move beyond this phase, that's okay! You'll be plugged in and can keep your finger on the pulse of this very fascinating project.
+
+### Best Newcomer Resources:
+
+\[Doc Link]
+
+This is a resource list of documentation and tools it will be best to familiarize yourself with as you dip your toes into the community. Don't feel like you need to read through everything before you're allowed to start interacting with the community or asking questions. Though by the time you truly start trying to contribute, you should be familiar with these basic concepts and resources.
+
+### Great Phase 1 Quests:
+
+* **Look through the Best Newcomer Resources list** above, and explore any other \[Documentation](link to an "All Docs Index" webpage) that looks interesting to you.
```

Comment:
Should mention the forums here as well. Lots of good content on there

Reply:
The discourse forums are explained in the Our Platforms doc which is in the Best Resources doc. Do you think it should also be accessible from this list? It's such a mess, I hesitate to put it front and center.

Reply:
Oh ok, if its there it should be good enough